docs/design/liqonet_tunEndCreator.md

```diff
@@ -0,0 +1,25 @@
+# TunnelEndpointCreator-Operator
+## Overview
+The TunnelEndpointCreator is part of the network module and runs as a deployment. It reacts to **Advertisements Custom Resources (CR)**
```

Comment:
If you write in bold CR here, you should write in bold all CRs, also in the other .md files.